ui/src/main/webapp/src/app/components/reports/application-index/aggregated-statistics.service.ts

```diff
@@ -0,0 +1,35 @@
+import {Injectable} from "@angular/core";
+import {Http} from "@angular/http";
+import {Observable} from "rxjs";
+import {AbstractService} from "../../../services/abtract.service";
+import {Constants} from "../../../constants";
+import {IssueCategoryModel} from "../../../generated/tsModels/IssueCategoryModel";
+
+@Injectable()
+export class AggregatedStatisticsService extends AbstractService {
+    private static AGGREGATED_MANDATORY_INCIDENTS_URL = 'aggregatedstats/aggregatedMandatoryIncidents';
+    private static AGGREGATED_CATEGORIES_URL = 'aggregatedstats/aggregatedCategories';
+    private static AGGREGATED_JAVA_PACKAGES_URL = 'aggregatedstats/aggregatedJavaPackages';
+
+    constructor(private _http: Http) {
+        super();
+    }
+
+    getAggregatedCategories(executionId: number): Observable<Map<IssueCategoryModel,number>> {
+        return this._http.get(`${Constants.GRAPH_REST_BASE}/reports/${executionId}/${AggregatedStatisticsService.AGGREGATED_CATEGORIES_URL}`)
+            .map(res => res.json())
+            .catch(this.handleError);
+    }
+ 
+    getMandatoryIncidents(executionId: number): Observable<Map<string, number>> {
+        return this._http.get(`${Constants.GRAPH_REST_BASE}/reports/${executionId}/${AggregatedStatisticsService.AGGREGATED_MANDATORY_INCIDENTS_URL}`)
+            .map(res => res.json())
+            .catch(this.handleError);
+    }
+
+    getAggregatedJavaPackages(executionId: number): Observable<Map<string, number>> {
```

Comment:
Return types from those methods are not real Map instances, but objects with string properties and number values.
TypeScript type declaration for those is little bit cucumbersome, but I guess you could use something like this:

```typescript
export interface Dictionary {
    [index: string]: number;
}
```
or even better

```typescript
export interface Dictionary<key, value> {
    [index: key]: value;
}
```
and in Observable use this: 
```typescript
Observable<Dictionary<string, number>>
```

(if it doesn't work with proposed `Dictionary` type, just leave it with `any`, I will fix it later, I think we have it in multiple places)